cmake/Allowlist.cmake

```diff
@@ -0,0 +1,32 @@
+
+if(__caffe2_allowlist_included)
+  return()
+endif()
+
+set(__caffe2_allowlist_included TRUE)
+
+set(CAFFE2_ALLOWLISTED_FILES)
+if(NOT CAFFE2_ALLOWLIST)
```

Comment:
I think this should be consistent with here: https://github.com/pytorch/pytorch/blob/master/CMakeLists.txt#L452

I don't know how/whether this is used by any other OSS projects - looks it's introduced for caffe2 selective build: https://github.com/facebookarchive/caffe2/issues/285. Updating external visible flags probably should be mentioned in release note. cc: @dzhulgakov  who might have more historical context.

Reply:
If this is used by external projects, let's not modify it for now. Once all of the easy replacements are done, regular contributors (e.g. like me) can handle more complicated cases like this that have dependencies on other projects. I don't think such task is appropriate for new or first-time contributors.